### ISSUE TITLE
rx problem

### ISSUE THREAD

**Issue by serdartncl** [Dfinitski/N7DDC-ATU-100-mini-and-extended-boards#57]
hi. firstly sorry for my english. i have a problem for my atu100. It was working well before but now when i on rx i can hear just noise but its like static noise. but tuner is works on tx and sending rf. i use with my qrp device under 2 watts. atu100 not modified for under 5 watt. if i tx up to 5 watt may be this problem solve or can i have to do anything?

**Comment by Dfinitski** [comment]
Hi.
How it is possible to use ATU-100 with no modification ? The original board does not start a tuning process with power below 5 Watt.

**Comment by serdartncl** [comment]
i dont need to process now. i just using for power and swr reading. i using for mono band and dipole for this

**Comment by Dfinitski** [comment]
Ok, I got you.
I don't know what could happened, but most likely you are hearing interferences from the display wires, if the FW fersion below 3.2.
Also, if your tuner uses power supply from DC/DC converter , it can be a source of interference too.

**Comment by serdartncl** [comment]
bir kaç gündür böyle bir problem yoktu. step up dc regülatör kullanıyorum farklı bir adaptör ile deneyip tekrardan bilgilendireceğim. teşekkür ederim

**Comment by serdartncl** [comment]
There was no such problem for a few days. I am using a step up dc regulator, I will try it with a different adapter and inform it again. thank you

**Comment by serdartncl** [comment]
I guess, as you said, the problem is in the regulator. I've never thought of it before, thanks again :)

**Comment by pimol** [comment]
There are difference if the ATU power is on or off? It seems a problem with
some connector...

I2OIM

Il giorno gio 6 mag 2021 alle ore 23:27 serdartncl ***@***.***>
ha scritto:

**Comment by serdartncl** [comment]
Yes ita different. When atu100 is off i didnt hear static noise. When i turn on its coming
